src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/application-card.component.html

```diff
@@ -68,32 +68,40 @@ <h5 *ngIf="userRole === role.provider">
     </div>
     <ng-container *ngIf="isMobileView; then actionsBlock"></ng-container>
   </mat-card-content>
-  <mat-card-content class="card-block card-block__buttons" *ngIf="userRole === role.provider && !application.isBlockedByProvider">
-    <ng-container *ngIf="application.workshop.competitiveSelection; else approveButtonBlock">
-      <ng-container *ngIf="application.status === applicationStatuses.Pending">
-        <button class="btn" (click)="acceptForSelection.emit(application)" [ngClass]="isApproveBtnHidden && 'hidden'">
-          <p>{{ 'BUTTONS.ACCEPT_TO_COMPETITIVE_SELECTION' | translate }}</p>
-        </button>
+
+  <ng-container *ngIf="userRole === role.provider && !application.isBlockedByProvider; else elseTemplate">
```

Comment:
It is better to call the template by a more specific name, `elseTemplate` is too general.